flarebyte/bubblegum-ui-textarea
# Migrate project to Elm 0.19 and whisker for code generation

# Summary of the change

- after elm-upgrade
- Normalize the code structure with baldrick-elm version 0.7.0 See https://github.com/flarebyte/baldrick-elm/releases
- Normalize the code structure with baldrick-elm version 0.7.0 See https://github.com/flarebyte/baldrick-elm/releases
- delete .message
- .message in gitignore
- Uses patch 1.1.1 for bubblegum-entity
- Normalize the code structure with baldrick-elm version 0.9.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.9.0 See https://github.com/flarebyte/baldrick-elm/releases
- Merge technical design document
- Restructure markdown technical design
- Generate WidgetPackageJson
- convert csv files to json
- convert json to yaml
- Adds support for json schema in vscode
- schema assistance for yaml files
- preparing to generate vocabulary
- preparing to generate vocabulary
- Generate generator script
- remove dashes from name in vocabulary
- Generate Vocabulary.elm with whisker
- draft of generating vocabulary helper
- Generate properly VocabularyHelper
- Adds template for key-description
- beautify demo too
- KeyDescription.elm generated by whisker
- Generate WidgetDocData
- generate Widget Create tests
- Generate correctly WidgetCreateTests
- Migrate IsoLanguage to be generated by whisker
- Convert transaltion csv to yaml
- Skeleton of translation
- generate internationalisation
- Deleted deprecated script files
- Externalize normalize.sh
- Makefile delegate to whisker-norm
- Code generation delegates more to baldrick-reserve
- Adds small markdown about acessibility and i18n
- Adds list of generated files
- Generate CODE_BASE.md
- Fixes schema for code-base
- rename filename to filepath
- toward full generation
- Generate app.js and css
- generate docs from demo
- get the latest css
- Adds accessibility checking for demo with tota11y
- make doc as part of generate
- How CODE_BASE.md should look
- new generate.sh
- Normalize the code structure with baldrick-elm version 0.10.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.11.0 See https://github.com/flarebyte/baldrick-elm/releasesNormalize the code structure with baldrick-elm version 0.12.0 See https://github.com/flarebyte/baldrick-elm/releases
- Normalize the code structure with baldrick-elm version 0.13.0 See https://github.com/flarebyte/baldrick-elm/releases
- skeleton for playwright
- big test is using playwright
- Test bubblegum widget with playwright
- smarter playwright.config.ts
- delete workflow
- Normalize the code structure with baldrick-elm version 0.13.0 See https://github.com/flarebyte/baldrick-elm/releases
## Code check

-   [x] `make ready` does not show any concerning issues

## Type of change


-   [x] Safe refactoring (non-breaking change which improves the code or
    documentation)


    functionality to not work as expected)

## Motivation and context

-   [ ] improve user experience
-   [x] improve consistency
-   [ ] improve security
-   [x] improve documentation
-   [x] improve code
-   [ ] reduce risk for unfamiliar tasks
-   [x] automate repetitive tasks

## How Has This Been Tested

-   [x] Automated browser tests
